tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph/graph_component.css

```diff
@@ -13,12 +13,64 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+:host {
+  overflow-y: auto;
+}
+
+.consumers-container {
+  padding-bottom: 5px;
+  overflow-x: auto;
+  white-space: nowrap;
+}
+
+.consumer-section {
+  display: block;
+  margin: 5px 0;
+}
+
 .graph-structure-container {
-  text-align: center;
+  font-size: 12px;
+  overflow-y: auto;
+  white-space: nowrap;
+}
+
+.inputs-container {
+  border-bottom: 1px solid rgb(0, 0, 0, 0.12);
+  margin-top: 5px;
+  overflow-x: auto;
+  padding-bottom: 0;
+  white-space: nowrap;
 }
 
 .input-op-container {
-  display: inline-table;
+  display: inline-block;
+  padding-top: 5px;
+}
+
+.input-op-name {
+  display: inline-block;
+  overflow-wrap: anywhere;
+  text-decoration: underline;
+}
+
+.input-op-output-slot {
+  display: inline-block;
+}
+
+.input-op-section {
+  border-right: 1px solid rgb(0, 0, 0, 0.12);
```

Comment:
:( wish we can use proper colors defined in our theming (tensorflow color scheme). No AI required for now. debugger has not been following that.